engo_glfw.go

```diff
@@ -317,7 +352,8 @@ func SetVSync(enabled bool) {
 	}
 }
 
-func init() {
```

Comment:
Because it isn't needed. You already know the assets so why don't you just load it in the setup? But then maybe you have a point and maybe `Preload` makes sense.
These are just a set of propositions.